Demo/utils.py

```diff
@@ -0,0 +1,42 @@
+import os
+from disposableredis import DisposableRedis
+
+
+REDIS_MODULE_PATH_ENVVAR = 'REDIS_MODULE_PATH'
+REDIS_PATH_ENVVAR = 'REDIS_PATH'
+REDIS_PORT_ENVVAR = 'REDIS_PORT'
+
+
+def execute_query(redis_graph, query_desc, query):
+    print(query_desc)
+    print("query: {query}".format(query=query))
+    print("execution plan:\n{plan}".format(plan=redis_graph.execution_plan(query)))
+    query_res = redis_graph.query(query)
+    print(query_res.pretty_print())
```

Comment:
pretty_print should print,
no need to print its return value.

Reply:
done